kornia/models/detector/rtdetr.py

```diff
@@ -0,0 +1,140 @@
+import warnings
+from typing import Optional
+
+import torch
+from torch import nn
+
+from kornia.contrib.models.rt_detr import DETRPostProcessor
+from kornia.contrib.models.rt_detr.model import RTDETR, RTDETRConfig
+from kornia.contrib.object_detection import ObjectDetector, ResizePreProcessor
+from kornia.core import rand
+
+
+class RTDETRDetectorBuilder:
+    """A builder class for constructing RT-DETR object detection models.
+
+    This class provides static methods to:
+        - Build an object detection model from a model name or configuration.
+        - Export the model to ONNX format for inference.
+    """
+
+    @staticmethod
+    def build(
+        model_name: Optional[str] = None,
+        config: Optional[RTDETRConfig] = None,
+        pretrained: bool = True,
```

Comment:
Tuple ?

Reply:
what?

Reply:
sorry my bad, that was supposed to be in another line where image size is only defined by a single int